connectors/src/api/webhooks/webhook_slack.ts

```diff
@@ -371,7 +377,86 @@ const _webhookSlackAPIHandler = async (
         }
         break;
       }
+      case "channel_created": {
+        const teamId = req.body.team_id;
```

Comment:
Let's extract this in a dedicated function, this switch/case becomes very hard to parse.